docs/eva_grace.md

```diff
@@ -3,3 +3,17 @@
 All nf-core pipelines have been successfully configured for use on the GRACE cluster at the [Max Planck Institute for Evolutionary Anthropology (MPI-EVA)](http://eva.mpg.de).
 
 To use, run the pipeline with `-profile eva_grace`. You can further with optimise submissions by specifying which cluster queue you are using e,g, `-profile eva_grace`. This will download and launch the [`eva_grace.config`](../conf/eva_grace.config) which has been pre-configured with a setup suitable.
+
+## Instructions
+
+Before running `nextflow run...` make sure to activate either the singularity (older pipelines, e.g. nf-core/eager v2) or apptainer (recent pipelines) with:
```

Comment:
This seems odd and really shouldn't matter?

And it's normally a bad idea relying on slurm carrying over the configuration, a `beforeScript "module load apptainer"` is probably a good idea.

Reply:
We run from the headnode rather than submitted jobs... (where containers get pulled), and singularity is installed globally (for some reason) so it's not necessary for the time being

And until we have eager into DSL2 we can't really switich full to loading apptainer currently :(